src/batch/src/executor/monitor/stats.rs

```diff
@@ -11,27 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
+use std::collections::HashMap;
+use std::sync::RwLock;
+use std::time::Duration;
+
 use prometheus::core::{AtomicU64, GenericCounterVec};
 use prometheus::{
     exponential_buckets, histogram_opts, register_histogram_with_registry,
     register_int_counter_vec_with_registry, Histogram, Registry,
 };
+use tokio::sync::mpsc::Sender;
 
-pub struct BatchMetrics {
-    pub row_seq_scan_next_duration: Histogram,
-    pub exchange_recv_row_number: GenericCounterVec<AtomicU64>,
+pub struct BatchTaskMetricsManager {
+    registry: Registry,
+    delete_queue_sender: Sender<DeleteRecord>,
+    exchange_recv_row_number: GenericCounterVec<AtomicU64>,
 }
 
-impl BatchMetrics {
+impl BatchTaskMetricsManager {
     pub fn new(registry: Registry) -> Self {
-        let opts = histogram_opts!(
-            "batch_row_seq_scan_next_duration",
-            "Time spent deserializing into a row in cell based table.",
-            exponential_buckets(0.0001, 2.0, 20).unwrap() // max 52s
-        );
-        let row_seq_scan_next_duration = register_histogram_with_registry!(opts, registry).unwrap();
-
```

Comment:
row_seq_scan metrics is removed?

Reply:
I move it in BatchTaskMetrics. Now we have two kinds of metrics:
- BatchMetrics
  exist in entire compute node lifecycle.
- BatchTaskMetrics
  exist in only a BatchExecution lifecycle. The recorded metrics will be deleted after execution done.

Reply:
But row_seq_scan should also be a per task metrics, isn't it? 🤔

Reply:
Seems it isn't. Because I look the use case of row_seq_scan and it didn't have any label like query_id or task_id.